components/retro-go/rg_display.c

```diff
@@ -927,7 +936,7 @@ void rg_display_clear(uint16_t color_le)
     size_t pixels = RG_SCREEN_WIDTH * RG_SCREEN_HEIGHT;
     uint16_t color = (color_le << 8) | (color_le >> 8);
 
-    lcd_set_window(0, 0, RG_SCREEN_WIDTH, RG_SCREEN_HEIGHT);
+    lcd_set_window(RG_SCREEN_MARGIN_LEFT, RG_SCREEN_MARGIN_TOP, RG_SCREEN_WIDTH, RG_SCREEN_HEIGHT);
```

Comment:
This will unfortunately break other devices; rg_display_clear must fill the whole panel even the unused parts because they can leak light. What is the issue when using `0, 0` offset on the qt py?

I'm fine with an #`ifdef target qtpy` for now, if you don't have time to investigate a universal solution!

Reply:
the qtpy display is 240x240 and is offet 80 pixels from the top. since RG_WIDTH and RG_HEIGHT is 240, this function will not fill the whole display - it ends up being about 3/4 because the 'top' 80 pixels arent visible.. if we wanted to really fill the whole framebuffer on chip, we'd have to hardcode this to be 320x240 (e.g. it isnt the same as the RG_WIDTH and RG_HEIGHT)!

Reply:
Thanks. I have a few ideas on how to make the function resolution agnostic but I don't want to waste your time suggesting random stuff so we'll just move ahead for now. I will most likely have to at least add an #ifdef fence for the mrgc-g32, though.